README.md

```diff
@@ -40,30 +40,7 @@ A brief video demonstration of building and using the device service can be foun
 # Learn More 
 
 ### General
-[Supported ONVIF features](./doc/ONVIF-protocol.md)  
```

Comment:
Shouldn't the above sections bee remove also and just reference the Docs. Guide links above are broken.